pom.xml

```diff
@@ -26,6 +26,7 @@
 		<module>agent</module>
 		<module>secret</module>
 		<module>ehcache</module>
+        <module>realtime</module>
```

Comment:
Could you please reformat the complete pom, we should also agree on a format, i use at the moment InteliJ Community with standard format. We also should add contributor guidelines https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/setting-guidelines-for-repository-contributors to it.